packages/frontapp/src/app/components/UserList/index.tsx

```diff
@@ -132,6 +143,13 @@ const UserList: FC<IProps> = ({ itemsTaken, sortBy }) => {
     }
   };
 
+  const navigate = useNavigate();
+  const handleGoBack = () => {
+    window.history.state.idx === 0
+      ? navigate(RoutesTypes.SEARCH)
+      : navigate(-1);
+  };
```

Comment:
As far as I know, in react defining functions in FC is bad practice, because they are created on each run. Use `useCallback` hook to ensure the function is only redefined when any of its dependencies change
Here docs https://stackoverflow.com/questions/46138145/where-should-functions-in-function-components-go